ansible_collections/arista/avd/plugins/action/eos_designs_documentation.py

```diff
@@ -121,8 +123,19 @@ def main(self, validated_args: dict, task_vars: dict, result: dict) -> dict:
 
         if output.digital_twin:
             content = {str(key).replace("_", "-"): value for key, value in asdict(output.digital_twin).items() if value is not None}
+            # Adjust formatting of the dumped Digital Twin artifact
+            match output.digital_twin:
+                case ACTDigitalTwin():
+                    serialized_content = "---\n" + sub(
+                        r"^(\s*-|\s{4})",
+                        lambda match: (" " * 4 + text) if (text := match.group(0)).startswith("  -") else (" " * 2 + text),
+                        yaml.dump(content, Dumper=AnsibleDumper, sort_keys=False, indent=2, width=130),
+                        flags=MULTILINE,
+                    )
+                case _:
+                    serialized_content = (yaml.dump(content, Dumper=AnsibleDumper, sort_keys=False, indent=2, width=130),)
```

Comment:
Why is this a tuple?
```suggestion
                    serialized_content = yaml.dump(content, Dumper=AnsibleDumper, sort_keys=False, indent=2, width=130)
```
Also, since you are also using `serialized_content` in the case above, you could move this out of the match block:
```suggestion
                    serialized_content = base_yaml
```

Reply:
For the tuple - i must have wrapped it into something else, then removed but haven't cleaned everything properly.
Moved base_yaml outside of the match/case.
Thanks!